index.html

```diff
@@ -938,18 +982,18 @@ <h2>Generic DID URL Parameters</h2>
         </p>
 
         <p>
-  Adding a DID parameter to a DID URL means that the parameter becomes part of
-  an identifier for a resource (the DID document or other). Alternatively, the
-  DID resolution and the DID URL dereferencing processes can also be influenced
-  by passing options to a <a>DID resolver</a> that are not part of the DID URL.
+  Adding a DID parameter to a <a>DID URL</a> means that the parameter becomes part of
+  an identifier for a resource (the <a>DID document</a> or other). Alternatively, the
+  <a>DID resolution</a> and the <a>DID URL dereferencing</a> functions can also be influenced
+  by passing input options to a <a>DID resolver</a> that are not part of the DID URL.
   Such options could for example control caching or the desired encoding of a
   resolution result. This is comparable to HTTP, where certain parameters could
   either be included in an HTTP URL, or alternatively passed as HTTP headers
   during the dereferencing process. The important distinction is that DID
   parameters that are part of the <a>DID URL</a> should be used to specify
-  <i>what resource is being identified</i>, whereas <a>DID resolver</a> options
+  <i>what resource is being identified</i>, whereas <a>DID resolution</a> options
   that are not part of the <a>DID URL</a> should be use to control <i>how that
```

Comment:
Same as above: These changes are editorial fixes outside this pull request and therefore I am not inclined to include them in order to keep pull this pull request smaller and more focused. Please submit a separate pull request to change this language instead of addressing it here.